doc/sphinx/getting_started.rst

```diff
@@ -0,0 +1,28 @@
+Getting Started
+===============
+
+This section aims to give you the knowledge necessary to use GPflow on small-to-medium projects,
+without necessarily going too much into the mathematical and technical details. We do not try to
+teach the theory behind Gaussian Processes. If you need a deeper understanding of the technical
+details or advanced features of GPflow please see our `User Guide`_.
```

Comment:
```suggestion
details or advanced features of GPflow please see our `User Guide`.
```

Reply:
Fergus, that `_` turns it into a link. It's not a mistake. :-)

Reply:
Hah ok good to know. :) I could have sworn I saw it rendered as a '_' in the browser, but it looks fine now. The only small thing I spot on the 'Getting Started' page now is: "Case Study: \(CO_2\) Levels"